tests/infrastructure/certificates.go

```diff
@@ -197,19 +202,14 @@ func getCertFromSecret(secretName string) []byte {
 	return nil
 }
 
-// getPodsCertIfSynced returns the certificate for all matching pods once all of them use the same certificate
-func getPodsCertIfSynced(labelSelector string, namespace string, port string) (cert []byte, err error) {
-	certs, err := tests.GetCertsForPods(labelSelector, namespace, port)
-	if err != nil {
-		return nil, err
-	}
+func hasSyncedCerts(certs [][]byte) []byte {
```

Comment:
`hasSyncedCerts` and it returns a cert? It does not sound intuitive to me.

Reply:
will change according https://github.com/kubevirt/kubevirt/pull/11763#discussion_r1573847961